sections/security/secureserver.md

```diff
@@ -5,9 +5,9 @@
 
 ### One Paragraph Explainer
 
-Using services like the [Let'sEncrypt](https://letsencrypt.org/) certificate authority providing __free__ ssl/tls certificates, you can easily obtain a certificate to secure your application. Node.js frameworks like [Express](http://expressjs.com/) (based on the core `https` module) support ssl/tls based servers easily, so the configuration can be done in a few lines of additional code.
+Using services such as [Let's Encrypt](https://letsencrypt.org/), a certificate authority who provide __free__ SSL/TLS certificates, can help encrypt the communication of your applications. Node.js frameworks like [Express](http://expressjs.com/) (based on the core `https` module) supports SSL/TLS, which can be implemented in a few lines of code.

```

Comment:
I’d change a few small details like the following:
- Stick to the providing for Let’sEncrypt
- supports -> support because it’s a plural (frameworks like express ... support)

Reply:
@BrunoScheufler I'm unclear on what you mean by 'Stick to the providing for Let'sEncrypt, do you mean change it back to what it was before e.g Using services like the, or removing the space

Reply:
Sorry being unclear, I meant sticking to the old text, but if I look at it now, we could also rephrase the sentence to something like: "Using services such as Let'sEncrypt, a certificate authority which provides __free__ SSK/TLS certificates..."

Reply:
Sure, I made the updates, let me know if you have any changes in mind